### YamlMime:JavaMember
uid: "com.azure.resourcemanager.dataprotection.models.UserFacingError.innerError*"
fullName: "com.azure.resourcemanager.dataprotection.models.UserFacingError.innerError"
name: "innerError"
nameWithType: "UserFacingError.innerError"
members:
- uid: "com.azure.resourcemanager.dataprotection.models.UserFacingError.innerError()"
  fullName: "com.azure.resourcemanager.dataprotection.models.UserFacingError.innerError()"
  name: "innerError()"
  nameWithType: "UserFacingError.innerError()"
  summary: "Get the innerError property: InnerError Inner Error."
  syntax: "public InnerError innerError()"
  returns:
    description: "the innerError value."
    type: "<xref href=\"com.azure.resourcemanager.dataprotection.models.InnerError?alt=com.azure.resourcemanager.dataprotection.models.InnerError&text=InnerError\" data-throw-if-not-resolved=\"False\" />"
type: "method"
metadata: {}
package: "com.azure.resourcemanager.dataprotection.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-dataprotection:1.0.0-beta.1
